vng_api_common/audittrails/viewsets.py

```diff
@@ -55,6 +55,8 @@ def create_audittrail(self, status_code, action, version_before_edit, version_af
             applicatie_weergave=applicatie_weergave,
             actie=action,
             actie_weergave=CommonResourceAction.labels.get(action, ''),
+            gebruikers_id=self.request.jwt_auth.payload['user_id'],
+            gebruikers_weergave=self.request.jwt_auth.payload['user_representation'],
```

Comment:
Since the field is optional, we should be a bit more lenient here I think

```suggestion
            gebruikers_weergave=self.request.jwt_auth.payload.get('user_representation', ''),
```